Makefile

```diff
@@ -151,6 +151,7 @@ MDLINKCHECK := $(shell command -v markdown-link-check 2> /dev/null)
 # Get link check tool
 get-linkcheck:
 ifndef MDLINKCHECK
+	sudo apt-get update
```

Comment:
This should go after the next log line and ideally combined together with install, like: `sudo apt-get update && sudo apt-get install npm`.